pkg/sql/codegen/pai/pai.go

```diff
@@ -40,3 +40,18 @@ func getPAIPredictCmd(ir *ir.PredictStmt, session *pb.Session) (string, error) {
 	return fmt.Sprintf(`pai -name prediction -DmodelName="%s" -DinputTableName="%s" -DoutputTableName="%s" -DfeatureColNames="%s" -DappendColNames="%s"`,
 		ir.Using, ir.TmpPredictTable, ir.ResultTable, strings.Join(flds, ","), strings.Join(flds, ",")), nil
 }
+
+// CleanupPAIModel can drop saved PAI model
+func CleanupPAIModel(ir *ir.TrainStmt, session *pb.Session) error {
+	db, err := database.OpenAndConnectDB(session.DbConnStr)
+	if err != nil {
+		return err
+	}
+	if ir.Estimator == "kmeans" || ir.Estimator == "randomforests" {
+		_, err := db.Exec(fmt.Sprintf("drop offlinemodel if EXISTS %s", ir.Into))
+		if err != nil {
```

Comment:
Very few suggestions:
1. Keep the consistency of capitalization within a SQL.
2. Merge L51&L52.